internal/project/project.go

```diff
@@ -410,6 +410,11 @@ func getCodeBlocksFromFile(path string) (document.CodeBlocks, error) {
 func getCodeBlocks(data []byte) (document.CodeBlocks, error) {
 	identityResolver := identity.NewResolver(identity.DefaultLifecycleIdentity)
 	d := document.New(data, identityResolver)
+
+	if f, _ := d.Frontmatter(); f != nil && f.Runme.Session.ID != "" {
```

Comment:
Suggestion: To be completely safe in the case the API changes, I recommend:

```suggestion
	if f, err := d.Frontmatter(); err == nil && f != nil && f.Runme.Session.ID != "" {
```